lib/resources/dstu2/medication_statement.yaml

```diff
@@ -0,0 +1,287 @@
+                 "code": {
+                   "text": "q4hr"}},
+               "asNeededCodeableConcept": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "25064002",
+                    "display": "headache (finding)",
+                    "userSelected": false}],
+               "text": "headache"},
+               "route": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "26643006",
+                    "display": "Oral route (qualifier value)",
+                    "userSelected": false}],
+                 "text": "Oral"},
+               "quantityQuantity": {
+                 "value": 325,
+                 "units": "mg",
+                 "system": "http://unitsofmeasure.org",
+                 "code": "mg"}}]}
+  action: create
+  children:
+
+  - name: text
+    required: 'No'
+    cardinality: 0..1
+    type: string
+    description: Free text dosage information as reported about a patient's medication use.
+    example: '{"text": "2, Oral, 2x/Wk, first dose 05/05/15 10:00:00 EDT"}'
+
+  - name: timing
+    required: 'No'
+    cardinality: 0..1
+    type: Timing
+    description: The timing schedule for giving the medication to the patient.
+    example: |
+               {"timing": {
+                 "repeat": {
+                   "boundsPeriod": {
+                     "start": "2014-11-03T14:38:00.000-05:00"},
+                   "count": 5},
+                 "code": {
+                   "coding": [
+                     {"system": "http://hl7.org/fhir/v3/vs/GTSAbbreviation",
+                      "code": "TID",
+                      "display": "TID"}],
+                   "text": "TID"}}
+    children:
+
+    - name: code
+      required: 'No'
+      cardinality: 0..1
+      type: CodeableConcept
+      description: A code for the timing pattern.
+      example: |
+                 {"code": {
+                   "coding": [
+                     {"system": "http://hl7.org/fhir/v3/vs/GTSAbbreviation",
+                       "code": "TID",
+                       "display": "TID"}],
+                   "text": "TID"}}}
+      url: http://hl7.org/fhir/DSTU2/datatypes-definitions.html#Timing.code
+      binding:
+        description: Code for a known / defined timing pattern.
+        terminology:
+        - display: TimingAbbreviation
+          system: http://hl7.org/fhir/timing-abbreviation
+        - display: SNOMED CT
+          system: http://snomed.info/sct
+
+    - name: repeat
+      required: 'No'
+      cardinality: 0..1
+      type: Element
+      description: A set of rules that describe when the event should occur.
+      example: |
+                 {"repeat": {
+                    "boundsPeriod": {
+                      "start": "2014-11-03T14:38:00.000-05:00"},
+                    "count": 5}}
+      children:
+
+      - name: bounds[X]
```

Comment:
I think we should only be using the [x] notation when we are actually supporting more than one option. Since we only support Period, this should be boundsPeriod